### ISSUE TITLE
Default/Stable PROJECT_TAGS failure end of May 2022

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#1133]
This was caused by `ICUB` master requiring YARP 3.7.0 without merging https://github.com/robotology/robotology-superbuild/pull/1129 before.

**Comment by traversaro** [comment]
After https://github.com/robotology/robotology-superbuild/pull/1129, the failure should be limited to `ROBOTOLOGY_ENABLE_TELEOPERATION` and a few other components. The failure is:
~~~
2022-05-26T08:06:51.2138293Z [1/15] Building CXX object modules/Utils/CMakeFiles/UtilityLibrary.dir/src/Utils.cpp.o
2022-05-26T08:06:51.2138667Z FAILED: modules/Utils/CMakeFiles/UtilityLibrary.dir/src/Utils.cpp.o 
2022-05-26T08:06:51.2140831Z /usr/share/miniconda3/envs/test/bin/x86_64-conda-linux-gnu-c++  -I/home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/include -isystem /home/runner/work/robotology-superbuild/robotology-superbuild/build/install/include -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -isystem /usr/share/miniconda3/envs/test/include -O3 -DNDEBUG -std=gnu++17 -MD -MT modules/Utils/CMakeFiles/UtilityLibrary.dir/src/Utils.cpp.o -MF modules/Utils/CMakeFiles/UtilityLibrary.dir/src/Utils.cpp.o.d -o modules/Utils/CMakeFiles/UtilityLibrary.dir/src/Utils.cpp.o -c /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp
2022-05-26T08:06:51.2143090Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp: In function 'bool YarpHelper::getDoubleFromSearchable(const yarp::os::Searchable&, const string&, double&)':
2022-05-26T08:06:51.2144257Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp:95:17: error: 'class yarp::os::Value' has no member named 'isDouble'
2022-05-26T08:06:51.2144765Z    95 |     if (!value->isDouble())
2022-05-26T08:06:51.2144997Z       |                 ^~~~~~~~
2022-05-26T08:06:51.2145691Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp:101:21: error: 'class yarp::os::Value' has no member named 'asDouble'
2022-05-26T08:06:51.2146202Z   101 |     number = value->asDouble();
2022-05-26T08:06:51.2146433Z       |                     ^~~~~~~~
2022-05-26T08:06:51.2147273Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp: In function 'bool YarpHelper::getYarpVectorFromSearchable(const yarp::os::Searchable&, const string&, yarp::sig::Vector&)':
2022-05-26T08:06:51.2148374Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp:134:31: error: 'class yarp::os::Value' has no member named 'isDouble'
2022-05-26T08:06:51.2148970Z   134 |         if (!inputPtr->get(i).isDouble() && !inputPtr->get(i).isInt())
2022-05-26T08:06:51.2149259Z       |                               ^~~~~~~~
2022-05-26T08:06:51.2149996Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp:134:63: error: 'class yarp::os::Value' has no member named 'isInt'; did you mean 'isInt8'?
2022-05-26T08:06:51.2150612Z   134 |         if (!inputPtr->get(i).isDouble() && !inputPtr->get(i).isInt())
2022-05-26T08:06:51.2150912Z       |                                                               ^~~~~
2022-05-26T08:06:51.2151172Z       |                                                               isInt8
2022-05-26T08:06:51.2152187Z /home/runner/work/robotology-superbuild/robotology-superbuild/src/walking-teleoperation/modules/Utils/src/Utils.cpp:140:38: error: 'class yarp::os::Value' has no member named 'asDouble'
~~~

Basically, we would need the fixes in https://github.com/robotology/walking-teleoperation/pull/97 available in walking-teleoperation master, or we switch PROJECT_TAGS Stable to use walking-teleoperation. @S-Dafarra @GiulioRomualdi as I guess you are busy with demo/conference and a lot is still going in that repo, if it is ok for you I would backport  https://github.com/robotology/walking-teleoperation/pull/97 on walking-teleoperation master and release walking-teleoperation 1.2.1 with YARP 3.7 compatibility.

**Comment by traversaro** [comment]
Default/Stable has been fixed: https://github.com/robotology/robotology-superbuild/actions/runs/2396297456 .
